frame.php

```diff
@@ -96,7 +96,7 @@ function startProview(
         previewStyle, 
         clear) {
         const referenceLinksArray = reference_link.split('\r\n').map(link => {
-            const [url, caption] = link.split(':');
+            const [url, caption] = link.split('::');
```

Comment:
@rohansharmasitoula can we use markdown syntax for link data capturing?
\[Label\]\(url\)

Reply:
changed 
@rakeshprabhu